src/dotnet/commands/dotnet-new/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.New
+{
+    internal static class LocalizableStrings
+    {
+        public static const string AppName = "dotnet new";
+
+        public static const string AppFullName = ".NET Initializer";
+
+        public static const string AppDescription = "Initializes empty project for .NET Platform"; 
+
+        public static const string CreateFailedBecauseProjectExists = "Creating new {0} project failed, project already exists.";
+
+        public static const string CreateFailedBecauseContentExists = "Creating new {0} project failed, directory already contains {1}";
+
+        public static const string CreateFailed = "Creating new {0} project failed.";
```

Comment:
We'll get into the nitt-gritty of translation and adjust. For now we are focusing on the calling convention from CLI into the localization objects. I agree we'll likely need to give help, but it's unclear what that format will be.